README.md

```diff
@@ -99,6 +99,31 @@ Depending on your parent pom, some of the `${jenkins.version}` properties may al
 
 If your code actually writes code against classes in any of these dependencies, remove the `<scope>test</scope>` entry for the corresponding block(s).
 
+Define Custom Classpath For Script Loading
+==============================
```

Comment:
Ah, I missed this earlier - this should probably go in the GroovyDoc of `JenkinsPipelineSpecification`, for consistency. That's the main "how-to" location with all the nuances & information about how to use it:

> <img width="664" alt="Screen Shot 2019-11-25 at 10 23 18 AM" src="https://user-images.githubusercontent.com/25155847/69559110-f6d20680-0f6e-11ea-9a72-11f61c4c2902.png">
> -- https://javadoc.io/static/com.homeaway.devtools.jenkins/jenkins-spock/2.0.1/com/homeaway/devtools/jenkins/testing/JenkinsPipelineSpecification.html
 
I would add this documentation to either side of the "Metaprogramming" bit at the end, and be sure to update the table of contents to link down to it, as well. Rendering code examples correctly in JavaDoc/GroovyDoc can be tricky; luckily there are lots of examples in there to refer to.